src/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <title>Rejestracja</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+</head>
+<body>
+<form class="container">
+    <h1>Rejestracja</h1>
+    <div class="mb-3">
+        <label for="firstName" class="form-label">Imię</label>
+        <input type="text" class="form-control" id="firstName" placeholder="Wpisz imię">
+    </div>
+    <div class="mb-3">
+        <label for="lastName" class="form-label">Nazwisko</label>
+        <input type="text" class="form-control" id="lastName" placeholder="Wpisz nazwisko">
+    </div>
+    <fieldset class="row mb-3">
+        <legend class="col-form-label col-sm-2 pt-0">Płeć</legend>
+        <div class="col-sm-10">
+            <div class="form-check">
+                <input class="form-check-input" type="radio" name="gender" id="female" checked>
+                <label class="form-check-label" for="female">
+                    Kobieta
+                </label>
+            </div>
+            <div class="form-check">
+                <input class="form-check-input" type="radio" name="gender" id="male">
+                <label class="form-check-label" for="male">
+                    Mężczyzna
+                </label>
+            </div>
+        </div>
+
+    </fieldset>
+    <div class="mb-3">
+        <label for="email" class="form-label">Adres email</label>
+        <input type="email" class="form-control" id="email" placeholder="Wpisz adres email">
+    </div>
+    <div class="mb-3">
+        <label for="pass" class="form-label">Hasło</label>
+        <input type="password" class="form-control" id="pass" placeholder="Hasło" aria-describedby="passwordHelpBlock">
+    </div>
+    <div id="passwordHelpBlock" class="form-text">
+        Min. 8 znaków
+    </div>
+    <div class="row">
+        <div class="col">
+            <label for="inputCity" class="form-label">Miasto</label>
+            <input type="text" class="form-control" id="inputCity">
+        </div>
+        <div class="col">
+            <label for="inputState" class="form-label">Województwo</label>
+            <select id="inputState" class="form-select">
+                <option selected>Wybierz...</option>
```

Comment:
Zmień to tak by `Wybierz...` nie było opcją dostępną do wyboru i by nie było widoczne na rozwijanej liście